backend/sftp/sftp.go

```diff
@@ -903,6 +944,34 @@ func parseHash(bytes []byte) string {
 	return strings.Split(string(bytes), " ")[0] // Split at hash / filename separator
 }
 
+// Parses the byte array output from the SSH session
+// returned by an invocation of df into
+// the disk size, used space, and avaliable space on the disk, in that order.
+// Only works when `df` has output info on only one disk
+func parseUsage(bytes []byte) (int64, int64, int64) {
+	lines := strings.Split(string(bytes), "\n")
+	if len(lines) < 2 {
+		return -1, -1, -1
+	}
+	split := regexp.MustCompile(" +").Split(lines[1], -1)
```

Comment:
If you are splitting on whitespace only then I would have thought `strings.Fields` would work here wouldn't it?

Reply:
Sorry, I didn't actually know about `strings.Fields`. I've changed it to use `strings.Fields` now.